src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -44,7 +44,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <!-- Optimize the framework using the crossgen2 tool. Crossgen2 is not currently supported on s390x or ppc64le or armv6 or loongarch64. -->
     <CrossgenOutput Condition=" '$(TargetArchitecture)' == 's390x' OR '$(TargetArchitecture)' == 'armv6' OR '$(TargetArchitecture)' == 'ppc64le' OR '$(TargetArchitecture)' == 'loongarch64' ">false</CrossgenOutput>
     <!-- VMR doesn't produce valid crossgen for the host/build machine https://github.com/dotnet/source-build/issues/3793 -->
-    <CrossgenOutput Condition=" '$(DotnetBuildVertical)' == 'true' AND '$(Crossbuild)' == 'true' ">false</CrossgenOutput>
+    <CrossgenOutput Condition=" '$(DotNetBuild)' == 'true' AND '$(Crossbuild)' == 'true' ">false</CrossgenOutput>
```

Comment:
Needs attention, /cc @jtschuster's issue

Reply:
What issue is that?

Reply:
https://github.com/dotnet/source-build/issues/3793

Reply:
LGTM, thanks for the heads up.